db/re/skill_db.yml

```diff
@@ -720,25 +721,25 @@ Body:
     AfterCastActDelay: 500
     Duration2:
       - Level: 1
-        Time: 5500
+        Time: 2500
       - Level: 2
-        Time: 8500
+        Time: 5500
       - Level: 3
-        Time: 11500
+        Time: 8500
       - Level: 4
-        Time: 14500
+        Time: 11500
       - Level: 5
-        Time: 17500
+        Time: 14500
       - Level: 6
-        Time: 20500
+        Time: 17500
       - Level: 7
-        Time: 23500
+        Time: 20500
       - Level: 8
-        Time: 26500
+        Time: 23500
       - Level: 9
-        Time: 29500
+        Time: 26500
       - Level: 10
-        Time: 32500
+        Time: 29500
```

Comment:
Pretty sure the duration here should be 3s*level just like pre-renewal and this was just a measuring error on my part due to latency.